WordPress/Classes/ViewRelated/Stats/JetpackLoginViewController.swift

```diff
@@ -2,18 +2,21 @@ import Foundation
 import UIKit
 import WordPressShared
 
-/// A view controller that presents a Jetpack login form
+/// A view controller that presents a Jetpack login form.
 ///
 class JetpackLoginViewController: UIViewController {
 
     // MARK: - Constants
 
     fileprivate let jetpackInstallRelativePath = "plugin-install.php?tab=plugin-information&plugin=jetpack"
     fileprivate let jetpackMoreInformationURL = "https://apps.wordpress.com/support/#faq-ios-15"
-    fileprivate let blog: Blog
+    var blog: Blog
 
     // MARK: - Properties
 
+    // Defaulting to stats because since that one is written in ObcC we don't have access to the enum there.
+    var presenter: JetpackLoginViewControllerPresenter = .stats
```

Comment:
Since this is more about the contents of the message, and it doesn't need to know what is presenting it, what about  a different name:

```swift
var promptType: JetpackLoginPromptType = .stats
```

Reply:
Agreed, I'll update it :)